### PR TITLE
[config][SlashIndicator] Update slash configs

### PR DESCRIPTION
**Pull request by nxqbao**
#### Update slash configs
- Validator slash:
    - Tier 1: 100 blocks
    - Tier 2: 500 blocks, slash amount: 1,000 RON
    - Tier 3: 100 more blocks, slash amount: 1,000 RON
- Bridge Slash:
    - Remove jail time
    - Remove Tier 2 self-slash, i.e., nodes only lose the bridge reward and validator rewards
- Voting Slash:
    - Slash amount: 1.000 RON

#### Upgrade logic
- `ValidatorSet`
- `SlashIndicator`

### CODE REVIEW

File: src/upgrades/230406-update-slash-conditions.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  const timestampBefore = blockBefore.timestamp;
+  const proposalExpiryTimestamp = timestampBefore + 3600 * 24 * 10; // expired in 10 days
+
+  console.log(2020);
+  console.log(proposalExpiryTimestamp);
+  console.log(instructions.map(() => slashProxyAddress));
+  console.log(instructions.map(() => 0));
+  console.log(instructions);
+  console.log(instructions.map(() => 1_000_000));
+
```
Comment (ducthotran2010) [inline src/upgrades/230406-update-slash-conditions.ts:72]:
```suggestion

```

**Comment by nxqbao** [comment]
Validating state after executing log

```
Logs:
  GAContract 0x946397deDFd2f79b75a72B322944a21C3240c9c3
  TOContract 0x98D0230884448B3E2f09a177433D60fb1E19C090
  SlashContract 0xEBFFF2b32fA0dF9C5C8C5d5AAa7e8b51d5207bA3
  TOLogicAddr 0x877A15cD0bf1667BaaB50b4C85FD79125c47b7B3
  governor 0xe880802580a1fbdeF67ACe39D1B21c5b2C74f059

  ===== PRE-CHECK =====
  missingVotesRatioTier1 1000
  missingVotesRatioTier2 3000
  jailDurationForMissingVotesRatioTier2 57600
  skipBridgeOperatorSlashingThreshold 50
  unavailabilityTier1Threshold 50
  unavailabilityTier2Threshold 150
  slashAmountForUnavailabilityTier2Threshold 10000000000000000000000
  jailDurationForUnavailabilityTier2Threshold 57600
  bridgeVotingThreshold 86400
  bridgeVotingSlashAmount 10000000000000000000000
  =====================

  ===== POST-CHECK =====
  missingVotesRatioTier1 1000
  missingVotesRatioTier2 3000
  jailDurationForMissingVotesRatioTier2 0
  skipBridgeOperatorSlashingThreshold 50
  unavailabilityTier1Threshold 100
  unavailabilityTier2Threshold 500
  slashAmountForUnavailabilityTier2Threshold 1000000000000000000000
  jailDurationForUnavailabilityTier2Threshold 57600
  bridgeVotingThreshold 86400
  bridgeVotingSlashAmount 1000000000000000000000
  ======================
```

File: src/upgrades/230406-update-slash-conditions.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    { from: governor, log: true },
+    'proposeProposalForCurrentNetwork',
+    // RoninGovernanceAdminContract.proposeProposalForCurrentNetwork(
+    //   expiryTimestamp,
+    //   targets,
+    //   values,
+    //   datas,
+    //   gasAmounts,
+    //   Ballot.VoteType.For
+    // );
```
Comment (ducthotran2010) [inline src/upgrades/230406-update-slash-conditions.ts:79]:
```suggestion
```

File: src/upgrades/230406-update-slash-conditions.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    proposalExpiryTimestamp,
+    instructions.map(() => slashProxyAddress),
+    instructions.map(() => 0),
+    instructions,
+    instructions.map(() => 1_000_000),
+    VoteType.For
```
Comment (ducthotran2010) [inline src/upgrades/230406-update-slash-conditions.ts:85]:
```suggestion
    proposalExpiryTimestamp, // expiryTimestamp
    instructions.map(() => slashProxyAddress), // targets
    instructions.map(() => 0), // values
    instructions, // datas
    instructions.map(() => 1_000_000), // gasAmounts
    VoteType.For // ballot type
```

**Comment by nxqbao** [comment]
Log of upgrading contract

```
    │   ├─ emit ProposalApproved(proposalHash: 0xd9c30228b82873b3fa17ca333998c4ea98ed661c354b53ab65c9bd9ef3f956d9)
    │   ├─ [5604] 0xEBFFF2b32fA0dF9C5C8C5d5AAa7e8b51d5207bA3::functionDelegateCall(0x0e1512ac00000000000000000000000000000000000000000000000000000000000003e80000000000000000000000000000000000000000000000000000000000000bb800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000032) 
    │   │   ├─ [4657] 0xB4E811e870928584e860e56A0D8656B0c77EAd55::setBridgeOperatorSlashingConfigs(1000, 3000, 0, 50) [delegatecall]
    │   │   │   ├─ emit BridgeOperatorSlashingConfigsUpdated(missingVotesRatioTier1: 1000, missingVotesRatioTier2: 3000, jailDurationForMissingVotesRatioTier2: 0, skipBridgeOperatorSlashingThreshold: 50)
    │   │   │   └─ ← ()
    │   │   └─ ← ()
    │   ├─ [13221] 0xEBFFF2b32fA0dF9C5C8C5d5AAa7e8b51d5207bA3::functionDelegateCall(0xd1737e27000000000000000000000000000000000000000000000000000000000000006400000000000000000000000000000000000000000000000000000000000001f400000000000000000000000000000000000000000000003635c9adc5dea00000000000000000000000000000000000000000000000000000000000000000e100) 
    │   │   ├─ [12037] 0xB4E811e870928584e860e56A0D8656B0c77EAd55::setUnavailabilitySlashingConfigs(100, 500, 1000000000000000000000, 57600) [delegatecall]
    │   │   │   ├─ emit UnavailabilitySlashingConfigsUpdated(unavailabilityTier1Threshold: 100, unavailabilityTier2Threshold: 500, slashAmountForUnavailabilityTier2Threshold: 1000000000000000000000, jailDurationForUnavailabilityTier2Threshold: 57600)
    │   │   │   └─ ← ()
    │   │   └─ ← ()
    │   ├─ [6295] 0xEBFFF2b32fA0dF9C5C8C5d5AAa7e8b51d5207bA3::functionDelegateCall(0x853af1b7000000000000000000000000000000000000000000000000000000000001518000000000000000000000000000000000000000000000003635c9adc5dea00000) 
    │   │   ├─ [5123] 0xB4E811e870928584e860e56A0D8656B0c77EAd55::setBridgeVotingSlashingConfigs(86400, 1000000000000000000000) [delegatecall]
    │   │   │   ├─ emit BridgeVotingSlashingConfigsUpdated(bridgeVotingThreshold: 86400, bridgeVotingSlashAmount: 1000000000000000000000)
    │   │   │   └─ ← ()
    │   │   └─ ← ()
    │   ├─ [7646] 0xEBFFF2b32fA0dF9C5C8C5d5AAa7e8b51d5207bA3::upgradeTo(0x056500E6028048dB7FCA81Ac307008A9042605f3) 
    │   │   ├─ emit Upgraded(implementation: 0x056500E6028048dB7FCA81Ac307008A9042605f3)
    │   │   └─ ← ()
    │   ├─ [11646] 0x617c5d73662282EA7FfD231E020eCa6D2B0D552f::upgradeTo(0x112119F52eC8760Dacc84907953F2baC6FE5107B) 
    │   │   ├─ emit Upgraded(implementation: 0x112119F52eC8760Dacc84907953F2baC6FE5107B)
    │   │   └─ ← ()
    │   ├─ emit ProposalExecuted(proposalHash: 0xd9c30228b82873b3fa17ca333998c4ea98ed661c354b53ab65c9bd9ef3f956d9, successCalls: [true, true, true, true, true], returnDatas: [0x, 0x, 0x, 0x, 0x])
```

**Review by ducthotran2010 — APPROVED** [review]

**Pull request merged**